src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -298,6 +298,13 @@ ov::Tensor merge_text_and_image_embeddings_llava(
 
     return merged_embeds;
 }
+
+// It's not possible to pass a GPU tensor from one model to another GPU
+// model on a different ov::Core instance.
+ov::Core singleton_core() {
+    static ov::Core core;
+    return core;
```

Comment:
I think it can be enough to create Core per pipeline? So, all sub-models within pipeline can share tensors of each other

Reply:
The problem is in `VisionEncoder` which isn't default constructible. As a result I have to initialize it in member initializer list. I could create a global instance of `ov::Core`, but singleton seems to be better because it won't create `ov::Core` if not needed.